qiskit/circuit/commutation_checker.py

```diff
@@ -20,17 +20,31 @@
 from qiskit.circuit import Qubit
 from qiskit.circuit.operation import Operation
 from qiskit.circuit.controlflow import CONTROL_FLOW_OP_NAMES
+from qiskit.circuit.library import XGate, YGate, ZGate
+from qiskit.circuit.library.generalized_gates.pauli import PauliGate
 from qiskit.quantum_info.operators import Operator
 
 _skipped_op_names = {"measure", "reset", "delay", "initialize"}
 _no_cache_op_names = {"annotated"}
 
+_rotation_gates = {
+    "rx": XGate(),
+    "ry": YGate(),
+    "rz": ZGate(),
+    "rxx": PauliGate("XX"),
+    "ryy": PauliGate("YY"),
+    "rzz": PauliGate("ZZ"),
+    "rzx": PauliGate("XZ"),
+}
+
 
 @lru_cache(maxsize=None)
 def _identity_op(num_qubits):
     """Cached identity matrix"""
     return Operator(
-        np.eye(2**num_qubits), input_dims=(2,) * num_qubits, output_dims=(2,) * num_qubits
+        np.eye(2**num_qubits),
```

Comment:
It seems your formatter is not set up the same way as Qiskit, maybe make sure to run black again with the local settings or update your lint to allow line lengths of 105 (see our `pyproject.toml`)